zh-Hans.lproj/Localizable.strings

```diff
@@ -297,11 +297,11 @@
 /* Help & Support */
 "MENU_HELP_GETSUPPORT_EMAIL" = "联系支持…";
 
-"SIDEBAR_CONTEXT_SHARE_COLLECTION_ELLIPSIS" = "共享…";
+"SIDEBAR_CONTEXT_SHARE_COLLECTION_ELLIPSIS" = "分享";
```

Comment:
Since there will be changes in the strings, what should I do at the moment? To change the translations to "Share" or "Share...", or wait for an update for the strings and create another pull request?

Reply:
This particular string "SIDEBAR_CONTEXT_SHARE_COLLECTION_ELLIPSIS" should be translated with the ellipsis. The non-ellipsis version is "SIDEBAR_CONTEXT_SHARE_COLLECTION". The app will be updated to look for _ELLIPSIS on macOS, and the other on iOS. The English strings file is correct, the app was not

Reply:
(I should clarify; I don't know what the rules are in Chinese (Simplified) for ellipsizing strings — if macOS does it, then I want to follow system convention. If not, then translate without the ellipsis. Thanks!)

Reply:
In Simplified Chinese the rule is the same as English. I've updated the translation, and thanks a lot for the clarification. 😊